server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EchoMethodContract.scala

```diff
@@ -132,14 +130,13 @@ trait EchoMethodContract {
   }
 
   @Test
-  def echoMethodShouldResponseOKWithRFC8621VersionAndSupportedMethod(): Unit = {
+  def echoMethodShouldRespondOKWithRFC8621VersionAndSupportedMethod(): Unit = {
```

Comment:
I think in scala if you have no params, the parenthesises `()` can be omitted ? (idem for other tests then if this is applicable)

Reply:
Yes, it need () to be method.If you remove it, you have no test suite.I tried

Reply:
the convention is that you use parenthesises if the method is doing side effects.

Reply:
> the convention is that you use parenthesises if the method is doing side effects.

Better at [MethodInvoking](https://docs.scala-lang.org/style/method-invocation.html)